package.json

```diff
@@ -15,20 +15,24 @@
     "clean": "rm -rf build/contracts",
     "compile": "truffle compile --all",
     "test:coverage": "SOLIDITY_COVERAGE=true npm run test",
+    "test:coverage:hardhat": "npx hardhat coverage",
```

Comment:
Just to confirm...

This npm script and `test:gas:hardhat` will not work right now because the tests rely on Truffle artifacts which cannot be used with hardhat without additional changes and that is expected right?

If this is the case, I suggest putting off adding these scripts until they actually work (i.e. after some test updates).

Reply:
Yes, that is correct. 

Alright, will remove the scripts and add it again with the test updates.